t/app/controller/admin.t

```diff
@@ -1470,6 +1470,7 @@ subtest "response priorities can be added" => sub {
         name => "Cat 1A",
         description => "Fixed within 24 hours",
         deleted => undef,
+        is_default => undef,
```

Comment:
Could we have a test setting it as well? I realise it's straightforward, but might as well to make sure it doesn't break in future :)